### PR TITLE
Use Flutter-recommended license format

### PR DESCRIPTION
**Pull request by advayDev1**
https://flutter.dev/docs/development/packages-and-plugins/developing-packages#adding-licenses-to-the-license-file

### CODE REVIEW

**Comment by SamMousa** [comment]
How are we making sure the file is up to date? Seems to me this file should be a build artifact instead of committed to our repo.. (something to check with legal?) ideally we just auto generate it for the app and only put our own in the license file.

**Comment by advayDev1** [comment]
This file is the part we /can't/ autogenerate.  Its third-party stuff people copied in to the repo.

For our client dependencies, flutter auto-generates a different LICENSE file for our end-user (click About the App -> Licenses).

File: LICENSE [hunk 1]
```diff
@@ -1,3 +1,5 @@
+ https://github.com/WorldHealthOrganization/app
```
Comment (advayDev1) [inline LICENSE:1]: This space is intentional - only way to sort it to the top of flutter's license list.

**Comment by advayDev1** [comment]
had to fix some formatting ptal @SamMousa

**Comment by SamMousa** [comment]
Did you squash your commits? I can't see the additional changes separately ;-)

**Comment by SamMousa** [comment]
There is the `applicationLegalese` property, which seems like it is intended for the the application's copyright notice: https://api.flutter.dev/flutter/material/LicensePage/applicationLegalese.html

Shouldn't we use that instead of the sort order hack for for LicenseRegistry?

**Comment by advayDev1** [comment]
@SamMousa good point!  Right now our About page points to our ToS and PP and general copyright, but we could put it there too.

**Pull request merged**